src/6_branch.js

```diff
@@ -67,6 +67,10 @@ var wrap = function(parameters, func, init) {
 			 * @type {function(?Error,?): undefined}
 			 */
 			var done = function(err, data) {
+				if (err) {
```

Comment:
@dmitrig01 I was thinking of that, maybe `setTimeout(next)` so that the error on the previous task could be handled before the next task fires, and then remove the call to `next()` on line 83. My reasoning for choosing the current route was that there didn't seem to be a way for a consumer to cancel subsequent tasks on an error, so this would basically force the issue. I'm on the fence about that, what are your thoughts?